### ISSUE TITLE
How to encrpy and then decrypt a video file using crpytoJS

### ISSUE THREAD

**Issue by VISHARAD17** [brix/crypto-js#392]
I am trying to encrypt a video file then store it in a local folder and then take it as a input and decrypt it and download the original file again

**Comment by shivam0831** [comment]
@VISHARAD17 did you get any solution?

**Comment by VISHARAD17** [comment]
Yes, I have implemented it by referring to one article online but I do not have its link.
You can check-out my repo [here](https://github.com/VISHARAD17/encryption_decryption) 
Thanks !

**Comment by shivam0831** [comment]
@VISHARAD17 Thanks for sharing your repo. I am working on react-native and your code is confusing for me. I just want to encrypt and decrypt video files from local storage.
